generators/server/templates/src/main/docker/couchbase/scripts/configure-node.sh

```diff
@@ -26,6 +26,9 @@ curl -u Administrator:password -X POST http://127.0.0.1:8091/settings/indexes -d
 echo "Setup bucket $BUCKET"
 curl -u Administrator:password -X POST http://127.0.0.1:8091/pools/default/buckets -d name=$BUCKET -d bucketType=couchbase -d ramQuotaMB=300 -dauthType=sasl
 
+echo "Setup bucket $BUCKET admin user"
+curl -u Administrator:password -X PUT http://127.0.0.1:8091/settings/rbac/users/local/$BUCKET -d name=$BUCKET -d password=password -d roles=bucket_admin%5B$BUCKET%5D
```

Comment:
@tchlyah could you add the same command for [kubernetes](https://github.com/jhipster/generator-jhipster/blob/master/generators/kubernetes/templates/db/_couchbase.yml#L101) and [openshift](https://github.com/jhipster/generator-jhipster/blob/master/generators/openshift/templates/db/_couchbase.yml#L149)?